ray_on_golem/server/services/golem/golem.py

```diff
@@ -104,4 +104,4 @@ def get_connection_uri(self, ip: str) -> URL:
 
     def get_ssh_proxy_command(self, connection_uri: URL) -> str:
         # Using single quotes for the authentication token as double quotes are causing problems with CLI character escaping in ray
-        return f"{self._websocat_path} -vv asyncstdio: {connection_uri}/22 --binary -H=Authorization:'Bearer {self._yagna_appkey}'"
+        return f"{self._websocat_path} asyncstdio: {connection_uri}/22 --binary -H=Authorization:'Bearer {self._yagna_appkey}'"
```

Comment:
@approxit please don't remove that flag, it makes debugging easier in case of issues